### DISCUSSION TITLE
4.6.5

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#180] [Announcements]
#### CHANGED 
- 标签页item添加左右边框
#### FIXED
- 修复自动切换下一P错误
- 修复漫游代理播放视频和搜索
- 修复评论备选方案未生效

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.5'>4.6.5</a>.</em>

**Comment by KennethWuWu** [comment]
大佬，看评论的时候，如果选择按时间排序评论区就是一片空白。

**Reply by ywmoyue** [reply]
这个问题4.6.6已修复
